benchmarks/css_asset_urls.rb

```diff
@@ -0,0 +1,36 @@
+# Benchmark file for the CssAssetUrls compiler
```

Comment:
I know this has merged, but I'd been stashing this sort of thing in `tools/benchmarks`

Reply:
Er, I commented on the wrong PR. Sorry y'all